transaction-api/src/main/java/com/example/auction/transaction/api/TransactionService.java

```diff
@@ -50,10 +54,15 @@
     default Descriptor descriptor() {
         return named("transaction").withCalls(
                 pathCall("/api/transaction/:id", this::submitDeliveryDetails),
-                pathCall("/api/transaction/:id", this::getTransaction)
-        ).withPathParamSerializer(UUID.class, PathParamSerializers.required("UUID", UUID::fromString, UUID::toString))
-                .withHeaderFilter(SecurityHeaderFilter.INSTANCE);
-
+                pathCall("/api/transaction/:id", this::getTransaction),
+                pathCall("/api/transaction?userId&status&pageNo&pageSize", this::getTransactionsForUser)
```

Comment:
is that `userId` meant to be in the queryString?
Hmmm, it looks like the quesryString declares 4 placeholders but using the wrong names.

If this was used by a third party client (non lagom client) it could be confusing for the developer.